### DISCUSSION TITLE
esLint: 'google' is not defined

### DISCUSSION THREAD

**Discussion by meet-bhalodiya-iverve** [visgl/react-google-maps#458] [Q&A]
I wanted to use the Circle component but when I use the code as mentioned in the Readme file.
I am getting bombarded by the google is not defined error and hence I am not able to use the `<Circle/>` component

I tried different things like:

Adding the reference to the Google Maps type definitions at the top:
`/// <reference types="google.maps" />`

after installing 
`npm install --save-dev @types/google.maps`

but nothing seems to be working
Can anyone help me what can I do to resolve this issue..?

**Comment by usefulthink** [comment]
In a typical typescript configuration this should just work, without needing to reference the types or anything like that, since typescript will just pick up all types from `node_modules/@types/*`. There are ways to configure typescript to not do that, so that might be a hint. You could maybe post the tsconfig.json you're using, so I can have a look.

You can check:
 - do you have the same issues with other parts of the maps library (for example, will it find the type for something like `const map: google.maps.Map | null = useMap();`)?
 - `@types/google.maps` should have been installed when installing @vis.gl/react-google-maps, so yo shouldn't need to install it yourself. Check if the directory `node_modules/@types/google.maps` exists. 

As for eslint, which versions and plugins are you using? 
Maybe you can append the relevant parts of your config here as well?
